R/eval_tidyselect.R

```diff
@@ -2,18 +2,20 @@
 ## determine which columns to span across
 ## ---------------------------------------
 eval_tidyselect_on_colvec <- function(x, column_vec){
-  span_col_select_function <- get(paste0("eval_tidyselect_on_colvec.",class(x)[1]),envir = asNamespace("tfrmt"))
-  span_col_select_function(x, column_vec = column_vec)
+  UseMethod("eval_tidyselect_on_colvec", x)
+  # span_col_select_function <- get(paste0("eval_tidyselect_on_colvec.",class(x)[1]),envir = asNamespace("tfrmt"))
+  # span_col_select_function(x, column_vec = column_vec)
 }
 
 #' @importFrom tidyselect eval_select
 #' @importFrom rlang !!!
 #' @importFrom dplyr expr
 eval_tidyselect_on_colvec.quosures <- function(x, column_vec){
-
   names(column_vec) <- column_vec
+  avli_x <- x %>%
+    keep(~as_label(.) %in% column_vec)
```

Comment:
two comments: 
1. imports need updating
2. do we need something like line 16 in the eval_tidyselect_on_colvec.quosure?

Reply:
I don't think we do. Cause the way tentative process works if there isn't a thing it will fail. The problem was it was failing when it should have passed for what was available